### ISSUE TITLE
Can I get vac ban for using skin changer?

### ISSUE THREAD

**Issue by Mikael-R** [namazso/nSkinz#239]
I would like to know if I can be banned or receive any punishment if I use this mod, preferably explain to me how the mod works

**Comment by namazso** [comment]
yes, of course you can. valve tends to only kick for detected skin changers, but they don't mind if some bans slip in.

> preferably explain to me how the mod works

read the source. it's placing some hooks here and there and changing variables
